deeplake/core/vectorstore/deeplake_vectorstore.py

```diff
@@ -87,6 +87,7 @@ def __init__(
             token (str, optional): Activeloop token, used for fetching user credentials. This is Optional, tokens are normally autogenerated. Defaults to None.
             overwrite (bool): If set to True this overwrites the Vector Store if it already exists. Defaults to False.
             verbose (bool): Whether to print summary of the dataset created. Defaults to True.
+            runtime (Dict, optional): Whether to store the dataset in the DeepLake Managed Database. Defaults to None.
```

Comment:
@adolkhan pls rephrase to:

runtime (Dict, optional): Parameters for creating the Vector Store in Deep Lake's Managed Tensor Database. Not applicable when loading an existing Vector Store. To create a Vector Store in the Managed Tensor Database, set `runtime = {"tensor_db": True}`.